omnibus/cookbooks/firezone/attributes/default.rb

```diff
@@ -521,3 +521,10 @@
 # Amount of time to sleep between connectivity checks, in seconds.
 # Default: 3600 (1 hour). Minimum: 60 (1 minute). Maximum: 86400 (1 day).
 default['firezone']['connectivity_checks']['interval'] = 3_600
+
+# === DEV ONLY ===
```

Comment:
Oo, probably not good to mix dev config and prod config. I don't think we ever want to disable the proxy requirement for prod?

Reply:
I found it useful to test the proxies without SSL using the package built by the CI

Reply:
I see. In that case, let's expose a new, documented configuration item:

```
default['firezone']['phoenix']['secure_cookies'] = true
```

We could likely add other cookie configuration options here as well down the road.